dcbr-web/src/components/OperationLocation.vue

```diff
@@ -59,7 +64,40 @@ import {mapState} from "vuex";
 export default {
   props: ["number"],
   data: () => ({
+    items: [
+      "Alberni-Clayoquot",
+      "Bulkley-Nechako",
+      "Capital",
+      "Cariboo",
+      "Central Coast",
+      "Central Kootenay",
+      "Central Okanagan",
+      "Columnbia Shuswap",
+      "Comox Valley",
+      "Cowichan Valley",
+      "East Kootenay",
+      "Fraser Valley",
+      "Fraser-Fort George",
+      "Islands Trust",
+      "Kitimat Stikine",
+      "Kootenay Boundary",
+      "Metro Vancouver",
+      "Mount Waddington",
+      "Nanaimo",
+      "North Okanagan",
+      "North Coast",
+      "Okanagan-Similkameen",
+      "Peace River",
+      "Gathet",
+      "Squamish-Lillooet",
+      "Strathcona",
+      "Sunshine Coast",
+      "Thompson-Nicola"
```

Comment:
Just linking back to my previous comment about the list of regions: we should really consolidate how they are retrieved since we currently have at least three places where we declare these values and there is a change we will have them mismatched...